### ISSUE TITLE
Organization: Profile

### ISSUE THREAD

**Issue by eazeredo** [bcgov/CONN-CCBC-portal#76]

**Comment by elliottSlee** [comment]
@eazeredo is this a duplicate of #30?

**Comment by eazeredo** [comment]
It looks like. 
Let's copy and add this description to there.

**Comment by eazeredo** [comment]
@zoeyli-46 and @elliottSlee , wondering if you evaluate the possibility of instead of having the dynamic toggle on the conditional questions, the page just shows all questions and the validation is made at the "Complete" action?

**Comment by elliottSlee** [comment]
@eazeredo I'm not sure I understand what you mean. We designed the conditional logic so the UX would be smoother, where applicants would only see questions pertaining to them.

**Comment by eazeredo** [comment]
An option is to replace this single date field for three dropdown fields: Year, Month, and Date.

I believe the date type field/component also allows to type the date, not only selecting from the datepicker. @marcellmueller @BryanGK

**Comment by zoeyli-46** [comment]
@marcellmueller cc: @MarsRomer 
Manual typing of a date is possible currently, though I do find choosing a month a bit hard. It seems there is only a drop down for year but not month. Is something like this from MUI X possible where it has both year and month picker options?
<img width="744" alt="image" src="https://user-images.githubusercontent.com/77090142/170574358-d1bab6b8-0be0-4afd-a608-8c652e5fc8f2.png">

**Comment by marcellmueller** [comment]
Yeah we could use a different custom datepicker package I'm sure. I was under the impression that we were using a styled [standard html date picker](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date) but we are actually already using a custom package [react-datepicker](https://www.npmjs.com/package/react-datepicker).

**Comment by zoeyli-46** [comment]
I think the requirement involves a date picker that is easy to choose a day, month, year that happened 50 years ago. That being a way to:
1. Manually type a date, formatted. can it have a placeholder or default value to show the format of YYYY/MM/DD
2. Quickly find a month 
3. Quickly find a year that is old (right now the date picker only lets me scroll down to 2017)

**Comment by marcellmueller** [comment]
@BryanGK what was the reasoning for using `react-datepicker` over a styled standard html date picker like the one in the [toolkit](https://button-inc.github.io/service-development-toolkit/@button-inc/bcgov-theme/index.html?path=/docs/components-datepicker--default)? 

I think the standard one that is used in the toolkit should be enough tbh.

**Comment by marcellmueller** [comment]
@zoeyli-46 the standard html datepicker used in the toolkit shows the date in the format of the language of the browser but it saves in the format YYYY/MM/DD.

**Comment by zoeyli-46** [comment]
Yes that toolkit one is great for the requirements, but the one on the https://ccbc-dev.apps.silver.devops.gov.bc.ca/ is not this one. 

Aware that you already flagged this in the above comment!

**Comment by elliottSlee** [comment]
@marcellmueller @BryanGK there's also this resource: https://reactdatepicker.com/#example-year-select-dropdown

@zoeyli-46 Do you think it's better to have year & month as drop downs or just year?

**Comment by zoeyli-46** [comment]
@elliottSlee having both year and month as drop-downs make it faster to find when they're looking for. In this use case, since organizations are likely to be old and won't be choosing a date in the near past or future, it may be easier for the user to find their year if there is a year picker with multi-columns. Is it possible to apply the [year picker](https://reactdatepicker.com/#example-year-picker) instead of the drop down when users click to change the year? 
<img width="200" alt="image" src="https://user-images.githubusercontent.com/77090142/170740325-255fc89d-7d55-4598-8d0f-0d92cb8e2a2e.png">


Additionally, on the current date picker its not visible that you can change the month as there is no down arrow next to the month. Could we use [this one](https://reactdatepicker.com/#example-year-select-dropdown) instead where it is more clear that it is an input field?
<img width="218" alt="image" src="https://user-images.githubusercontent.com/77090142/170740016-fd09d405-2269-4163-8c6f-478a188bf907.png">

**Comment by zoeyli-46** [comment]
@marcellmueller update: organization profile, location, authorized and alternate contact moved to the end of the form before declaration page.

**Comment by elliottSlee** [comment]
@marcellmueller @BryanGK would it be helpful to create another issue on Github for the re-ordering of pages?
